packages/vx-shape/src/types/base.ts

```diff
@@ -0,0 +1,22 @@
+import { D3Scale, PickD3Scale } from '@vx/scale';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type $TSFIXME = any;
```

Comment:
don't need to do anything ... but starting to think this is probably most useful for apps, not libs, where lib consumers prob just want to see `any` and not trace a `fixme` type.

Reply:
the only benefits now is it prevent `eslint` from automatically changing `any` to `unknown` which breaks a lot of things.